samples/crypto/rng/README.rst

```diff
@@ -7,14 +7,16 @@ Crypto: RNG
    :local:
    :depth: 2
 
+The RNG sample shows how to produce random numbers.
+
 Overview
 ********
 
-The RNG sample shows how to produce random numbers.
+The sample produces random values of 100-byte length.
 
-The sample produces random values of 100-byte length:
-   #. The Platform Security Architecture (PSA) API is initialized.
-   #. Five 100-byte random values are produced.
+1. First, the sample initializes the Platform Security Architecture (PSA) API.
+
+#. Then, it generates five 100-byte random values.
```

Comment:
Values is maybe a misnomer. Can number be used instead. The algorithm is R**N**G